utils/wgs_gene_coverage/get_gene_exon_intron.py

```diff
@@ -0,0 +1,86 @@
+import pandas as pd
+import os
+import sys
+import jupyter_black
+
+jupyter_black.load()
+
+## Read genome intron/exon coordinates (from parse_gft.R)
+def Read_Genome_BED():
```

Comment:
You should write the comments you have written above the functions as docstrings, e.g:

```
def Read_Genome_BED():
    """Read genome intron/exon coordinates (from parse_gft.R)"""
```